src/core/unittest/RecvBufferTest.cpp

```diff
@@ -100,21 +143,14 @@ struct RecvBuffer {
     ) {
         ASSERT_EQ(RecvBuf.ReadStart, ReadStart);
         ASSERT_EQ(RecvBuf.ReadLength, ReadLength);
-        int ChunkCount = 1;
-        QUIC_RECV_CHUNK* Chunk =
-            CXPLAT_CONTAINING_RECORD(
-                RecvBuf.Chunks.Flink,
-                QUIC_RECV_CHUNK,
-                Link);
-        ASSERT_EQ(Chunk->ExternalReference, ExternalReferences[0]);
-        while (Chunk->Link.Flink != &RecvBuf.Chunks) {
+
+        int ChunkCount = 0;
```

Comment:
Just to confirm my understanding, now it's possible for there to be zero chunks, because the app hasn't provided any? And that's why you made this change?

Reply:
Yes. There are two scenarios where we can have zero chunks:
- the app hasn't provided any (yet).
- all provided chunks have been used and relinquished.